client/app/scripts/superdesk-search/search.js

```diff
@@ -1215,6 +1215,17 @@
                         multiEdit.create(multi.getIds());
                         multiEdit.open();
                     };
+
+                    scope.createPackage = function() {
+                        packages.createPackageFromItems(multi.getItems())
+                        .then(function(new_package) {
+                            superdesk.intent('author', 'package', new_package);
+                        }, function(response) {
+                            if (response.status === 403 && response.data && response.data._message) {
```

Comment:
There are some package related validation that should be showed to the user and all those errors has 403 error code. 
For other critical errors we don't show any other message. In the application there is a mixed style re: critical errors. We have to decide what style to adopt and use it consistently.